src/jquery.fancytree.dnd5.js

```diff
@@ -502,13 +502,18 @@
 					break;
 			}
 
+			var dropMarkerRect;
```

Comment:
Please follow the project's conventions, i.e. group 'var's, use curly brackets, etc.